lib/job/resume.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+const date = require('date.js');
+
+/**
+ * Resumes a job into the MongoDB
+ * @name Job#
+ * @function
+ * @param when time for the new nextRunAt field
+ * @returns {Promise} instance of Job that is resumed
+ */
+module.exports = function(when) {
```

Comment:
How would we resume jobs with an interval set? Should we use `computerInterval`?

https://github.com/agenda/agenda/blob/b71d3eaecc9b6eb7af976a165b8fb27dcbebac24/lib/job/compute-next-run-at.js#L34